bootstrap/sql/org.postgresql.Driver/v001__create_db_connection_info.sql

```diff
@@ -196,6 +196,17 @@ CREATE TABLE IF NOT EXISTS ml_model_entity (
     UNIQUE (fullyQualifiedName)
 );
 
+CREATE TABLE IF NOT EXISTS container_entity (
```

Comment:
container is broader term here, is there anything else we can use that can be specific to object store

Reply:
renamed to objectstore_container_entity